client/js/store/statement/Statement.js

```diff
@@ -649,7 +649,7 @@ export default {
         })
         .catch(e => {
           console.error(e)
-          dplan.notify.error(Translator.trans('error.api.generic'))
+          dplan.notify.error('test')
```

Comment:
really?

Reply:
Thanks for noticing, done with [340ba33](https://github.com/demos-europe/demosplan-core/pull/4472/commits/340ba336c42ccacccd0b8ce811c4daf3891681cd)